.github/workflows/ci.yml

```diff
@@ -50,9 +50,6 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
-      - name: print node node-version
```

Comment:
I'd incorrectly merged this in with #1340. This reverts that change (it was only for debugging purposes).